scripts/find-broken-links

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# example usage: $ find-broken-links
+#  it's setup to use 1.11 docs Urls
+#  it will print the links that won't return 200 to stdout: <link> <http-status-code>
+
+files=$(git ls-files)
+for file in $files
+do
+  # skip files with links we don't care about
+  matchesNonCodeFile=$(echo $file | grep -Eo "(\.json|\.gif|\.png|\.svg|-test|-cy)")
```

Comment:
To cut out more noise, instead of telling which files to exclude we could tell which files to **include**.

For example: only search the `src` directory for JS files. This might be better.

Reply:
Perhaps you could pass in a path? So we could do `../../external` too?